src/routes/messages.js

```diff
@@ -40,8 +45,16 @@ module.exports = (() => {
     *     {
     *          "message": "Invalid/missing token"
     *     }
+    * 
+    * @apiError ChatOrMessageNotFound Message does not refer to a chat that exists.
+    *
+    * @apiErrorExample ChatOrMessageNotFound Response:
+    *     HTTP/1.1 404 Not Found
+    *     {
+    *          "message": "Given chat or message not found"
+    *     }
     */
-	router.post('/:mid/add-reaction', messageController.addReaction);
+	router.post('/:mid/reaction', messageController.addReaction);
 
```

Comment:
I didn't notice this earlier, but I changed /add-reaction to /reaction since we may want to use the same endpoint to have a user remove a reaction to a message in the future.